student.rb

```diff
@@ -1,6 +1,8 @@
 class Student < Person
   attr_accessor :classroom
 
+  belongs_to :classroom
+
```

Comment:
- [ ] Kindly add a method that serves as the 'belongs-to' side (a student belongs to a classroom).